composer/core/callback.py

```diff
@@ -92,150 +91,150 @@ def run_event(self, event: Event, state: State, logger: Logger) -> None:
-            state (State): The global state.
-            logger (Logger): The logger.
+            state (:class:`.State`): The global state.
+            logger (:class:`.Logger`): The logger.
         """
         del state, logger  # unused
         pass
 
     def batch_start(self, state: State, logger: Logger) -> None:
-        """Called on the :attr:`~.Event.BATCH_START` event.
+        """Called on the :attr:`.Event.BATCH_START` event.
 
         Args:
-            state (State): The global state.
-            logger (Logger): The logger.
+            state (:class:`.State`): The global state.
+            logger (:class:`.Logger`): The logger.
         """
         del state, logger  # unused
         pass
 
     def after_dataloader(self, state: State, logger: Logger) -> None:
-        """Called on the :attr:`~.Event.AFTER_DATALOADER` event.
+        """Called on the :attr:`.Event.AFTER_DATALOADER` event.
 
         Args:
-            state (State): The global state.
-            logger (Logger): The logger.
+            state (:class:`.State`): The global state.
+            logger (:class:`.Logger`): The logger.
         """
         del state, logger  # unused
         pass
 
     def before_train_batch(self, state: State, logger: Logger) -> None:
-        """Called on the :attr:`~.Event.BEFORE_TRAIN_BATCH` event.
+        """Called on the :attr:`.Event.BEFORE_TRAIN_BATCH` event.
 
         Args:
-            state (State): The global state.
-            logger (Logger): The logger.
+            state (:class:`.State`): The global state.
+            logger (:class:`.Logger`): The logger.
         """
         del state, logger  # unused
         pass
 
     def before_forward(self, state: State, logger: Logger) -> None:
-        """Called on the :attr:`~.Event.BEFORE_FORWARD` event.
+        """Called on the :attr:`.Event.BEFORE_FORWARD` event.
 
         Args:
-            state (State): The global state.
-            logger (Logger): The logger.
+            state (:class:`.State`): The global state.
+            logger (:class:`.Logger`): The logger.
         """
         del state, logger  # unused
         pass
 
     def after_forward(self, state: State, logger: Logger) -> None:
-        """Called on the :attr:`~.Event.AFTER_FORWARD` event.
+        """Called on the :attr:`.Event.AFTER_FORWARD` event.
 
         Args:
-            state (State): The global state.
-            logger (Logger): The logger.
+            state (:class:`.State`): The global state.
+            logger (:class:`.Logger`): The logger.
         """
         del state, logger  # unused
         pass
 
     def before_loss(self, state: State, logger: Logger) -> None:
-        """Called on the :attr:`~.Event.BEFORE_LOSS` event.
+        """Called on the :attr:`.Event.BEFORE_LOSS` event.
 
         Args:
-            state (State): The global state.
-            logger (Logger): The logger.
+            state (:class:`.State`): The global state.
```

Comment:
These were already auto-linking before (https://docs.mosaicml.com/en/stable/api_reference/composer.core.callback.html#module-composer.core.callback), so I dont think the changes are needed?